lms/envs/common.py

```diff
@@ -2569,6 +2569,7 @@
         "level_of_education",
         "mailing_address",
         "requires_parental_consent",
+        "account_privacy",
```

Comment:
Actually, @wajeeha-khalid please go ahead and make the move.  I have made the change on the mobile-dev sandbox, but you should make it for the PR so appropriate unit tests can be updated.